src/i18n/locales/en.json

```
@@ -290,6 +290,7 @@
   "groqApi": {
     "name": "Groq API",
     "llama2-70b-4096": "LLaMA2 70b",
+    "llama3-70b-8192": "LLama3 70b",
```

```suggestion
    "llama2-70b-4096": "Llama 2 70b",
    "llama3-70b-8192": "Llama 3 70b",
```

Could you change all the names like above? Meta has changed the official name from `LLaMA` to `Llama`

---

Sure, but I prefer not to touch the existing one in the same commit to make sure the scope is clear enough, would that be good to you?